internal/bean/implement/mail_client.go

```diff
@@ -24,18 +26,36 @@ func NewMailClient() bean.MailCLient {
 
 	dialer := gomail.NewDialer(host, port, username, password)
 
-	return &MailCLient{
-		from:     from,
-		dialer:   dialer,
+	return &MailClient{
+		from:   from,
+		dialer: dialer,
 	}
 }
 
-func (m *MailCLient) SendEmail(ctx context.Context, to string, subject string, body string) error {
+func (m *MailClient) SendEmail(ctx context.Context, to string, subject string, code string, context string, ttl time.Duration) error {
 	message := gomail.NewMessage()
-    message.SetHeader("From", m.from)
-    message.SetHeader("To", to)
-    message.SetHeader("Subject", subject)
-    message.SetBody("text/html", body)
 
-    return m.dialer.DialAndSend(message)
-}
\ No newline at end of file
+	contextMessage := fmt.Sprintf("Enter the following code to %s:", context)
+
+	body := fmt.Sprintf(`
+		<!DOCTYPE html>
+		<html>
+		<body>
+			<p>You have selected <strong>%s</strong> as your email address.</p>
+			<p>%s</p>
+			<h2 style="color: #4CAF50;">%s</h2>
+			<p>This code will expire %d minute after this email was sent.</p>
+			<hr>
+			<p><strong>Why you received this email.</strong></p>
+			<p>We require verification whenever an email address is updated.</p>
```

Comment:
We require verification whenever a password is updated or prior to a transaction.